spec/form_models/copy_vacancy_form_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.describe CopyVacancyForm, type: :model do
+  it 'copies the vacancy attributes to the form object' do
+    original_vacancy = build(:vacancy)
+
+    form_object = described_class.new(vacancy: original_vacancy)
+
+    expect(form_object.job_title).to eq(original_vacancy.job_title)
+    expect(form_object.starts_on).to eq(original_vacancy.starts_on)
+    expect(form_object.ends_on).to eq(original_vacancy.ends_on)
+    expect(form_object.expires_on).to eq(original_vacancy.expires_on)
+    expect(form_object.publish_on).to eq(original_vacancy.publish_on)
+  end
+
+  it 'updates the original vacancy with the users new preferences' do
```

Comment:
I reckon it might be good to add another test here to make sure the original `form_object` is mutated, so the user has access to their changes on error. That way we can see that this is an intended behaviour in case it falls out in any future refactorings